content/building-blocks/collaborate-and-share-your-work/use-github/git-lfs.md

```diff
@@ -12,103 +12,157 @@ aliases:
+{{% /tip %}}
 
-## Install Git LFS <!-- Provide your code in all the relevant languages and/or operating systems. -->
 
-Make sure that [Git is already installed](/building-blocks/configure-your-computer/statistics-and-computation/git/).
+### Installation 
 
-- Go to [git-lfs.github.com](https://git-lfs.github.com) and download directly. Or, if you use [`Brew`](/building-blocks/configure-your-computer/automation-and-workflows/commandline/#mac-users):
+- Ensure you have Git installed. If not, follow this [building block](/building-blocks/configure-your-computer/statistics-and-computation/git/) to set it up. 
 
-``` bash
+- Next, download Git LFS from [git-lfs.github.com](https://git-lfs.github.com) or use [`Brew`](/building-blocks/configure-your-computer/automation-and-workflows/commandline/#mac-users) for macOS users:
+
+{{% codeblock %}}
+```bash
 brew install git-lfs
 ```
+{{% /codeblock %}}
 
-If you used Brew, go to the next step. If you downloaded directly from the website, open the terminal and change the current working directory to the downloaded and unzipped folder of Git LFS. Then, install:
+- If you used Brew, go to the next step. If downloaded directly, open the terminal and change the current working directory to the downloaded and unzipped folder of Git LFS. Then, install:
 
+{{% codeblock %}}
 ``` bash
 ./install.sh
 ```
+{{% /codeblock %}}
 
 - Once installed, set up LFS for your account:
 
+{{% codeblock %}}
 ``` bash
 git lfs install
 ```
+{{% /codeblock %}}
 
-If it was successful, you should see ```Git LFS initialized```.
+- If it was successful, you should see the message ```Git LFS initialized```.
 
-## Usage
+### Explicit tracking
 
-Git LFS doesn't do anything autonomously for you. You need to **explicitly tell it which files to track**.
+Git LFS doesn't autonomously manage files: you must **explicitly tell it which files to track**.
 
+- To track a specific file, use:
+{{% codeblock %}}
 ``` bash
 git lfs track "largefile.png"
 ```
+{{% /codeblock %}}
 
-Or, to track a file type (if you don't want to manually specify every single file you wish to track):
+- Alternatively, to track multiple files of a specific type: 
 
+{{% codeblock %}}
 ``` bash
 git lfs track "*.png"
 ```
+{{% /codeblock %}}
 
 {{% warning %}}
-**Be careful!** Do not forget the quotes around your file name.
+Always enclose file names in quotes!
 {{% /warning %}}
 
+### Resuming workflow
+
 Now you can resume your usual Git workflow. You just need to make sure to track the `.gitattributes` file too.
 
+{{% codeblock %}}
 ``` bash
 git add .gitattributes
 ```
-Simply add your file(s), commit and push as you'd normally do!
+{{% /codeblock %}}
+
+Simply add your file(s), commit, and push as you'd normally do!
 
+{{% codeblock %}}
 ``` bash
 git add largefile.png
 git commit -m "Add large file"
 git push origin master
 ```
+{{% /codeblock %}}
 
-Note: To clone a repository and to pull the most recent changes before working on it, use:
+{{% tip %}}
```

Comment:
At this point you create a codeblock in a tipbox, which I think is not possible since it does not render correctly in the live preview. Maybe just at the information not as a tip but a normal section?